data/events/scripts/player.lua

```diff
@@ -86,6 +86,25 @@ function Player:onLookInShop(itemType, count)
 end
 
 function Player:onMoveItem(item, count, fromPosition, toPosition, fromCylinder, toCylinder)
+	-- Store Inbox
```

Comment:
I get what you are saying but we should keep as much in Lua as posible.
Having some codes in Lua can be faster than having them in c++.

Another reason for that is so we can modify things without needing to recompile everytime.

And gl using a 1.1 datapack with 1.3 or even 1.2 without doing changes.
Thats a thing we need to do, either as new prs are approved or once every year or so.

Either update your server files or stay where you are.